juristische-zitierweise-bern.csl

```diff
@@ -0,0 +1,494 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="expanded" demote-non-dropping-particle="never" default-locale="de-CH">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Juristische Zitierweise Bern (de-CH)</title>
```

Comment:
Made this change.

@rockawaybeach, I see that you created extensive documentation at https://csl.hundius.ch/. I was wondering if the original documentation by the University of Bern is also available online or in print.

And if you followed the guidelines of the law faculty of this university, it would probably be better to rename this style to "Universität Bern - Rechtswissenschaftliche Fakultät (German - Switzerland)".